platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -1125,11 +1134,12 @@ private <T> void updatePaths(
      *
      * @param mutation
      * 		The mutation list, can be null.
-     * @param <T>
+     * @param <LK> The key type held by the mutation.
```

Comment:
I guess, either a key (K) or a path?

Reply:
Renamed it to K1 and V1 to follow the convention

Reply:
Updated